fenix/app/src/main/java/org/mozilla/fenix/debugsettings/ui/DebugOverlay.kt

```diff
@@ -42,17 +44,21 @@ import org.mozilla.fenix.theme.FirefoxTheme
  *
  * @param navController [NavHostController] used to perform navigation actions.
  * @param drawerStatus The [DrawerStatus] indicating the physical state of the drawer.
+ * @param debugDrawerDestinations The complete list of [DebugDrawerDestination]s used to populate
+ * the [DebugDrawer] with sub screens.
  * @param onDrawerOpen Invoked when the drawer is opened.
  * @param onDrawerClose Invoked when the drawer is closed.
- * @param onBackButtonClick Invoked when the user taps on the back button in the app bar.
+ * @param onDrawerBackButtonClick Invoked when the user taps on the back button in the app bar.
  */
+@Suppress("LongParameterList")
```

Comment:
this can be removed now that https://github.com/mozilla-mobile/firefox-android/pull/5022 has merged (or feel free to leave as part of the clean up task in the linked PR)